doc/ipython-notebooks/gaussian_process/variational_classifier.ipynb

```diff
@@ -60,31 +111,51 @@
      "level": 2,
      "metadata": {},
      "source": [
-      "Brief Introduction"
+      "<a id='intro'>Brief Introduction</a> <a href='#toc'>[TOP]</a>"
      ]
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "In binary classification, we get binary labels in form of a column vector, $\\mathbf{y}\\in\\mathcal{Y}^n=\\{-1,+1\\}^n$, and features as a matrix, $\\mathbf{X}\\in\\mathcal{R}^{n\\times d}$, where $n$ is the number of data points and $d$ is the number of features.  A likelihood function $p(\\mathbf{y}|\\text{f},\\mathbf{X})=p(\\mathbf{y}|\\mathbf{f})$ is used to model data with labels, where a function $\\text{f}:\\mathcal{R}^{d}\\to \\mathcal{R} $ is drawn from a Gaussian Process prior and $\\text{f}(\\mathbf{X})=\\mathbf{f} \\in \\mathcal{R}^{n}$ is a column vector by applying f to each row of $\\mathbf{X}$. Note that the difference between f and $\\mathbf{f}$ is that f is a function while $\\mathbf{f}$ is an n-dimensional vector. $p(\\text{f})$ and $p(\\mathbf{f})$ are also different because $p(\\text{f})$ is a distribution in an infinite dimensional (function) space while $p(\\mathbf{f})$ is a distribution in an finite dimensional space.\n",
+      "In this notebook, we mainly focus on binary classification problems. It is not hard to extend to multi-classifcation problems.   Given a binary classification problem, binary labels and features are respectively defined as a column vector, $\\mathbf{y}\\in\\mathcal{Y}^n=\\{-1,+1\\}^n$, and a matrix, $\\mathbf{X}\\in\\mathcal{R}^{n\\times d}$, where $n$ is the number of data points and $d$ is the number of features.  A likelihood function $p(\\mathbf{y}|\\text{f},\\mathbf{X})=p(\\mathbf{y}|\\mathbf{f})$ is used to model data with labels, where a latent function $\\text{f}:\\mathcal{R}^{d}\\to \\mathcal{R} $ is drawn from a GP prior and $\\text{f}(\\mathbf{X})=\\mathbf{f} \\in \\mathcal{R}^{n}$ is a column vector by applying $\\text{f}$ to each row of $\\mathbf{X}$. Given data with labels, our goal is to train a GP classifier ($p(\\text{f}|\\mathbf{y},\\mathbf{X} $)) to fit the data well and predict new data points ($p(y_\\text{new}|\\mathbf{y},\\mathbf{X},\\mathbf{x}_\\text{new})$). Following sections coveres some detailed background of variational inference for GPC models.\n",
+      "\n",
+      "Note that the difference between $\\text{f}$ and $\\mathbf{f}$ is that $\\text{f}$ is a function while $\\mathbf{f}$ is a n-dimensional vector. The GP prior ($p(\\text{f})$) and $p(\\mathbf{f})=p(\\text{f}(\\mathbf{X}))$ are also different because $p(\\text{f})$ is a distribution in an infinite dimensional (function) space while $p(\\mathbf{f})$ is a distribution in a finite dimensional real space.\n",
```

Comment:
"The GP prior ($p(\\text{f})$) and"

dont but brackets around the math, just coma